package.json

```diff
@@ -127,7 +127,7 @@
       },
       {
         "command": "tomcat.war.browse",
-        "title": "browse"
+        "title": "open in browser"
       }
     ],
```

Comment:
Same

Reply:
My personal reason is that the Explorer view in visual studio code is limited. Lowercase context command looks more light and easy to use.

Reply:
OK, I will make them the first letter uppercase since I find almost all vscode context commands this kind of pattern.